components/logins/android/build.gradle

```diff
@@ -78,39 +61,12 @@ configurations {
     jnaForTest
 }
 
-protobuf {
-    protoc {
-        artifact = 'com.google.protobuf:protoc:3.11.4'
-    }
-    generateProtoTasks {
-        all().each { task ->
-            task.builtins {
-                java {
-                    option "lite"
-                }
-            }
-        }
-    }
-}
-
-// Needs to happen before `dependencies` in order for the variables
-// exposed by the plugin to be available for this project.
```

Comment:
Some of the changes in this file don't seem strictly related?

Reply:
Interestingly enough I just ran the gradle stuff when testing things and this was autogenerated? I'll try revert changes and see when it actually removes this.

Reply:
This all looks correct now 👍